cli/assets/CounterTemplate.t.sol

```diff
@@ -3,7 +3,7 @@ pragma solidity ^0.8.13;
 
 import "forge-std/Test.sol";
 
-contract ContractTest is Test {
+contract CounterTest is Test {
     function setUp() public {}
 
     function testExample() public {
```

Comment:
this should be something like `testIncrement` that calls `Counter.increment` and then uses assertEq to ensure it was successful